scripts/firedrake-install-conda.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
we don't usually use the .sh suffix

Reply:
I can remove the .sh suffix. But the linter is assuming this is a Python script, which it isn't. I think this makes the most sense as a shell script but can rewrite it in Python if need be (there's a [Conda Python API](https://conda.io/projects/conda/en/latest/api/index.html)).